code/mob/living/silicon/robot.dm

```diff
@@ -532,6 +532,12 @@
 							message = "<B>[src]</B> beep-bops at [M]."
 							break
 
+						if (istype(src.buckled, /obj/machinery/conveyor))
+							message = "<B>[src]</B> beep-bops and flips [himself_or_herself(src)] free from the conveyor."
+							src.buckled = null
+							if(isunconscious(src))
```

Comment:
smh why doesn't this just set paralyzed status

Reply:
<Insert long comment about strange implementation causing interesting and unintuitive behavior that produces interesting gameplay mechanics> 🤷